### PR TITLE
Feature/tcvp 1678

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Added new eticket, handwritten tickets without validating email addresses.  Tested resending email verification from staff portal.  Manually set 'email verified' flag to yes and tested they work as expected (waiting on citizen portal work on email verification).

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Sharkww — APPROVED** [review]

File: src/backend/TrafficCourts/Common/OpenAPIs/OracleDataAPI/v1_0/OracleDataApi.v1_0.json [hunk 1]
```diff
@@ -30,16 +30,16 @@
...
             "content": { "*/*": { "schema": { "type": "object" } } }
           },
           "405": {
             "description": "An invalid JJ Dispute status is provided. Update failed.",
             "content": { "*/*": { "schema": { "type": "object" } } }
           },
-          "404": {
-            "description": "JJDispute record not found. Update failed.",
+          "400": {
+            "description": "Bad Request.",
```
Comment (KDNeufeld) [inline src/backend/TrafficCourts/Common/OpenAPIs/OracleDataAPI/v1_0/OracleDataApi.v1_0.json:42]: Is there something in particular you changed in the OracleDataApi OpenAPI spec? As far as I can tell nothing has changed, except the usual random ordering of the responses.  Just wondering why this change in included in the PR.

File: src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs [hunk 2]
```diff
@@ -343,6 +343,60 @@ public async Task<IActionResult> CancelDisputeAsync(long disputeId, Cancellation
...
+    /// <response code="400">The request was not well formed. Check the parameters.</response>
+    /// <response code="401">Unauthenticated.</response>
+    /// <response code="403">Forbidden, requires dispute:submit permission.</response>
+    /// <response code="500">There was a server error that prevented the update from completing successfully.</response>
+    /// </returns>
+    [HttpPut("{disputeId}/resendEmailVerify")]
+    [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
```
Comment (KDNeufeld) [inline src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs:363]: I don't see 401 and 403 in the catch blocks of the ApiException. Are these returned? It looks like these would be converted to 500 errors.

File: src/backend/TrafficCourts/Workflow.Service/appsettings.Development.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "BaseUrl": "http://localhost:5010/"
   },
   "SmtpConfiguration": {
-    "Host": "localhost",
+    "Host": "apps.smtp.gov.bc.ca",
```
Comment (KDNeufeld) [inline src/backend/TrafficCourts/Workflow.Service/appsettings.Development.json:10]: I don't know if this should be checked in.  Maybe this should be part of UserSecrets?

For other projects, I've used gessnerfl/fake-smtp-server, a docker image that can be run within docker-compose and simulates an email server. All it does is just log the emails that were sent, but in a nice HTML website so we can see how emails with HTML content are rendered.

File: src/backend/TrafficCourts/Common/OpenAPIs/OracleDataAPI/v1_0/OracleDataApi.v1_0.json (hunk 1 above)
Comment (lfdgov) [reply src/backend/TrafficCourts/Common/OpenAPIs/OracleDataAPI/v1_0/OracleDataApi.v1_0.json:42]: no changes here except i generally re-import the JSON  and rebuild to see if any changes

File: src/backend/TrafficCourts/Workflow.Service/appsettings.Development.json (hunk 3 above)
Comment (lfdgov) [reply src/backend/TrafficCourts/Workflow.Service/appsettings.Development.json:10]: i put this in user secrets

File: src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs (hunk 2 above)
Comment (lfdgov) [reply src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs:363]: these would be caught before the body of the method was ever executed which is why they are not part of the other exceptions listed that might happen after the method was partway done executing.

**Review by KDNeufeld — APPROVED** [review]
Great work

**Pull request merged**